### DISCUSSION TITLE
指定局域网ip或mac设备直连

### DISCUSSION THREAD

**Discussion by jt8823** [Openwrt-Passwall/openwrt-passwall2#319] [Q&A] [answered]
请问 是在分流规则里设置还是访问控制里面设置啊
遇到了两个问题，第一是插件里面的实时流量监测不对
第二是我家智能设备有点多，最近很不稳定。

有办法指定ip绕过passwall核心吗？
或者直连？

**Comment by Ymy214** [comment]
添加一个分流：全部关闭就不走内核志杰转发给网关，访问控制里选择IP对应新建的分流即可

**Reply by jt8823** [reply]
这样对么?
![ ](https://i.miji.bid/2023/08/09/365cd120a68d2576248c8fec20c28250.png)
![ ](https://i.mij.rip/2023/08/09/246e973e03ca36b652264a1967679090.png)

**Comment by handsomeji** [comment, marked as answer]
https://github.com/xiaorouji/openwrt-passwall2/issues/273#issuecomment-1584462154
